### ISSUE TITLE
computed column yields `error: malformed array literal …`

### ISSUE THREAD

**Issue by ferdinandsalis** [calebmer/postgraphql#56]
I am trying to add a computed column (see below). I have tested the procedure and it returns exactly the row I ask for. However when execute it in a graphql query I get the following error `error: malformed array literal: "[34,24,null]"`.

```sql
-- Computed Columns

create function artwork_cover_url(artwork artwork) returns varchar as $$
  select i.uri as cover_url
    from artwork_image as ai
    join image as i
      on ai.image_id = i.id
    where ai.artwork_id = artwork.id
    order by ai.priority desc limit 1
$$ language sql stable
set search_path from current;
```
psql test output —

              artwork_cover_url |
----------------------------------------------
 2e705a23116e9512b2dcbd1768bde1893e648fde.jpg |
(1 row) |

**Comment by calebmer** [comment]
Thanks for the report! I'll take a look.

**Comment by ferdinandsalis** [comment]
These are the values the get passed to the procedure —

```
values:  [ anonymous {
    id: 1,
    artist_id: 1,
    name: 'La Danse a l’echarpe',
    dating: '1897',
    origin: null,
    medium: 'Öl auf Karton/Leinwand',
    inscription: 'Rechts unten monogrammiert',
    dimensions: [ 43, 55, null ],
    edition: null,
    meta:
     { expertise: null,
       exhibitions: [Object],
       provenience: [Object],
       publications: null },
    exposed: true,
    archived: false,
    created_at: 2014-07-06T08:48:40.000Z } ]
```
the offender is clearly the dimension column which is of type `numeric(6,2)[3]`
```
error: malformed array literal: "[43,55,null]"
```

**Comment by ferdinandsalis** [comment]
The problem appears to be that `json_populate_record` returns an json array `[1, 2]` whereas postgres understands `{1,2}`. So when creating the values object arrays (also multi-dimensional) need to be converted. Something like a prepareValues that traverses the object and converts array types to strings with `{` instead of `[`. Am I overthinking things or would that be right way to tackle this?

**Comment by ferdinandsalis** [comment]
This should be fixed with https://github.com/calebmer/postgraphql/pull/58

**Comment by ferdinandsalis** [comment]
Fix released with [`1.5.1`](https://github.com/calebmer/postgraphql/commit/d9c75a0247fbeda9a1961633ab8af5853246fa2e)
